test/blackbox-tests/test-cases/ctypes/deps-full-spec.t/run.t

```diff
@@ -0,0 +1,18 @@
+Build an example library as a DLL and set up the environment so that it looks
+like a system/distro library that can be probed with pkg-config and dynamically
+loaded.
+
+Then generate cstubs for it, build an executable that uses those cstubs, and
+run the executable that tests the library through the cstubs.
+
+  $ LIBEX=$(realpath "$PWD/../libexample")
+  $ DYLD_LIBRARY_PATH="$LIBEX" LD_LIBRARY_PATH="$LIBEX" PKG_CONFIG_PATH="$LIBEX/pkgconfig" PKG_CONFIG_ARGN="--define-prefix" dune exec ./example.exe
+  File "dune", line 14, characters 40-48:
+  14 |     "#include \"example.h\"\n#include \"%{foo_h}\"\n#include \"%{bar_h}\"\n#include \"baz.h\"\n#include \"%{dep:qux.h}\""))
+                                               ^^^^^^^^
+  Error: %{foo_h} isn't allowed in this position.
```

Comment:
I don't know what to look at to be able to fix this error

Reply:
Shouldn't this be `%{dep:foo.h}`? `

Reply:
No, I want to use the name of the dependency, this is on purpose.